### ISSUE TITLE
Feature request: Endless loop

### ISSUE THREAD

**Issue by RChadwick7** [derv82/wifite2#91]
I'd love to see a command line option to restart the scan and attack after it completes. This would be handy for headless devices, as well as getting a handshake from a device that isn't always available.

**Comment by ravenphreaker** [comment]
I agree one of the best things about besside-ng was the fact that it was a continous loop. A fire and forget sort of thing.back when wep was the major used envryption i would just load up besside with my awus036h 1 watt and walk away come back a half hour later and show my clients the favt that i had cracked there network along with everyone else and i only typed 6 lines in terminal (ifconfig wlan1 down, iw reg set BO (at the time), iwconfig wlan1 txpower 30, ifconfig wlan1 up, airmon-ng start wlan1, besside-ng -v wlan1mon). Anyway. Back to the case in point i think this would be a great idea.

**Comment by derv82** [comment]
Yep this is definitely a feature I want in Wifite. "Pillage" isn't nearly automated enough.

It would require a bunch of work to maintain state, select nearby routers, stop after X seconds, etc.

**Comment by RaduNico** [comment]
I've been working on implementing endless mode for Wifite and it has been almost done for a week now along with some general improvements. I am not very sure when I will be making the PR for this feature so for now it can be tested on my [fork](https://github.com/RaduNico/wifite2/tree/infinite_attack) on the infinite_attack branch. -inf runs wifite in infinite mode and the wait time is configurable with -p <time>.

Any feedback or comments is more than welcome. :)
